data/libs/systems/blessing.lua

```diff
@@ -83,26 +83,56 @@ Blessings.PlayerDeath = function(player, corpse, killer)
 	local hasAol = (player:getSlotItem(CONST_SLOT_NECKLACE) and player:getSlotItem(CONST_SLOT_NECKLACE):getId() == ITEM_AMULETOFLOSS)
 	local hasSkull = table.contains({ SKULL_RED, SKULL_BLACK }, player:getSkull())
 	local currBlessCount = player:getBlessings()
+	local blessCount = type(currBlessCount) == "table" and #currBlessCount or 0
+	local droppedItems = {}
```

Comment:
[luacheck] reported by reviewdog 🐶
value assigned to variable 'droppedItems' is unused

Raw Output:
data/libs/systems/blessing.lua:87:8: value assigned to variable 'droppedItems' is unused